woocommerce/navigation
# Add back to dashboard button

Fixes #64

Adds a button to navigate back to the wp admin dashboard if we're at the root of the navigation menu.

Note: Styling for this will be applied in https://github.com/woocommerce/navigation/pull/68

### Screenshots

<img width="197" alt="Screen Shot 2020-08-31 at 8 30 12 AM" src="https://user-images.githubusercontent.com/10561050/91720006-0128cd00-eb9f-11ea-8604-64a4d512db60.png">


### Testing

1. Check that at the root the button exists.
1. Check that it takes you to the correct location.
1. Check that when navigation to different levels within the navigation menu, the "WordPress Dashboard" button does not exist and instead the navigation back button exists.
